infra-templates/ipsec/15/rancher-compose.yml

```diff
@@ -1,7 +1,6 @@
 .catalog:
   name: "Rancher IPsec"
-  version: "0.2.1"
-  minimum_rancher_version: v1.6.11-rc1
```

Comment:
don't remove the min version, update to `v1.6.13-rc1`

Reply:
ok